pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/util/ConfigManager.java

```diff
@@ -68,7 +69,11 @@ public static synchronized ConfigManager getInstance() {
      * please use {@link #getInstance()}
      */
     public static Config parseConfig() {
-        return ConfigFactory.parseFile(TYPESAFE_CONFIG_FILE);
+        if (TYPESAFE_CONFIG_FILE != null) {
+            return ConfigFactory.parseFile(TYPESAFE_CONFIG_FILE);
+        }
+
+        return ConfigFactory.parseString(SecretManager.get("542678289221", "dss-configuration").orElseThrow());
```

Comment:
Obviously, we don't want to hardcode the project's and the secret's name. Do we want to have them as environment variables or CL arguments?

Reply:
Let's pass the secret name, the project id, and the version identifier via system properties.

How about `google.secret.project`, `google.secret.name`, and `google.secret.version`? `google.secret.version` should be optional and, if not set, should default to the literal string `latest`.

Reply:
SGTM